scan/lib/scan/runner.rb

```diff
@@ -93,7 +93,7 @@ def test_results
 
       reporter_options_generator = XCPrettyReporterOptionsGenerator.new(false, [], [], "", false)
       reporter_options = reporter_options_generator.generate_reporter_options
-      cmd = "cat #{@test_command_generator.xcodebuild_log_path} | xcpretty #{reporter_options.join(' ')} &> /dev/null"
+      cmd = "cat \"#{@test_command_generator.xcodebuild_log_path}\" | xcpretty #{reporter_options.join(' ')} &> /dev/null"
```

Comment:
Ahh @KrauseFx just told me it should be 
`cat #{@test_command_generator.xcodebuild_log_path.shellescape} | xcpretty #{reporter_options.join(' ')} &> /dev/null`
with the added require:
`require 'shellwords'`